include/saber/geometry/size.hpp

```diff
@@ -16,7 +16,7 @@ namespace saber::geometry {
 // 5. Build up binary free operators from class operators +=, etc
 
 template<typename T>
-class Size //: private EqualityOperators<Size<T>> // CRTP Curiously Recurring Template Pattern
+class Size // CRTP Curiously Recurring Template Pattern
```

Comment:
Comment doesn't make sense.
CRTP is when a class inherits from a template base class taking it's own type as template arg.